### ISSUE TITLE
Watch keeper on a baremetal cluster is not installing successfully

### ISSUE THREAD

**Issue by smukherjee16** [razee-io/WatchKeeper#249]
**Describe the bug**
We are trying to install the watch keeper on a baremetal cluster. We see a new document in clusters collection, but watch keeper pod has errors and keeps crashing. We tried version 0.6.6 and 0.7.3

We keep getting the same error 0.7.3
```
{“name”:“Index”,“hostname”:“rhm-watch-keeper-7d87d544-cjt4r”,“pid”:1,“level”:50,“msg”:“recieved unhandledRejection TypeError: Cannot read property ‘some’ of null\n    at KubeResourceMeta.hasVerb (/home/node/node_modules/@razee/kubernetes-util/lib/KubeResourceMeta.js:74:23)\n    at /home/node/node_modules/@razee/kubernetes-util/lib/kubeClass.js:221:37\n    at Array.filter (<anonymous>)\n    at KubeClass.getKubeResourcesMeta (/home/node/node_modules/@razee/kubernetes-util/lib/kubeClass.js:221:23)\n    at processTicksAndRejections (internal/process/task_queues.js:93:5)\n    at async Object.watch (/home/node/src/controllers/Watch.js:61:23)\n    at async main (/home/node/src/index.js:38:3)“,”time”:“2021-02-19T18:37:56.213Z”,“v”:0}
```

Error on 0.6.6
```
{"name":"Index","hostname":"rhm-watch-keeper-5645584d57-ss762","pid":18,"level":50,"err":{"message":"Cannot read property 'some' of null","name":"TypeError","stack":"TypeError: Cannot read property 'some' of null\n    at KubeResourceMeta.hasVerb (/home/node/node_modules/@razee/kubernetes-util/lib/KubeResourceMeta.js:74:23)\n    at /home/node/node_modules/@razee/kubernetes-util/lib/kubeClass.js:221:37\n    at Array.filter (<anonymous>)\n    at KubeClass.getKubeResourcesMeta (/home/node/node_modules/@razee/kubernetes-util/lib/kubeClass.js:221:23)\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async Object.watch (/home/node/src/controllers/Watch.js:61:23)\n    at async main (/home/node/src/index.js:38:3)"},"msg":"Unhandled promise rejection.","time":"2021-02-18T16:56:20.926Z","v":0}
```

If a fix will take sometime, can you please provide us with a workaround if available.

**Comment by alewitt2** [comment]
communicated over slack that this has been resolved with the updated watchkeeper version 0.7.4

**Comment by alewitt2** [comment]
from slack:

```

Hey good news from our side.. the watch keeper fix worked and now we are able to register a baremetal cluster 
thanks a lot for your fix on this so quick. We will upgrade to 0.7.4 now
```
